pubspec.yaml

```diff
@@ -19,6 +19,7 @@ dependencies:
   flutter:
     sdk: flutter
   flutter_launcher_icons: ^0.13.1
+  flutter_secure_storage: ^9.0.0
```

Comment:
Where do we need flutter secure storage in keypod?

Reply:
removed